prime-router/src/main/kotlin/Organization.kt

```diff
@@ -1,17 +1,56 @@
 package gov.cdc.prime.router
 
 /**
- * Organization represents a partner organization of the hub (eg. a sender or a receiver).
+ * Organization represents a partner organization of the hub. It has a jurisdiction.
  */
-data class Organization(
+open class Organization(
     val name: String,
-    val description: String? = null,
-    val clients: List<OrganizationClient> = emptyList(),
-    val services: List<OrganizationService> = emptyList(),
+    val description: String,
+    val jurisdiction: Jurisdiction,
+    val stateCode: String?,
+    val countyName: String?,
 ) {
-    init {
-        // init back-references
-        services.forEach { it.organization = this }
-        clients.forEach { it.organization = this }
+    enum class Jurisdiction {
+        FEDERAL,
+        STATE,
+        COUNTY
     }
-}
\ No newline at end of file
+
+    /**
+     * Validate the object and return null or an error message
+     */
+    fun consistencyErrorMessage(): String? {
+        return when (jurisdiction) {
+            Jurisdiction.FEDERAL -> {
+                if (stateCode != null || countyName != null)
+                    "stateCode or countyName not allowed for FEDERAL organizations"
+                else null
+            }
+            Jurisdiction.STATE -> {
+                if (stateCode == null || countyName != null)
+                    "stateCode required for STATE organizations"
+                else null
+            }
+            Jurisdiction.COUNTY -> {
+                if (stateCode == null || countyName == null)
+                    "stateCode and countyName required for COUNTY organizations"
+                else null
+            }
+        }
+    }
+}
+
+/**
+ * Organization with senders and receivers.
+ *
+ * Useful to put all the information about an org in single object or file.
+ */
+class DeepOrganization(
+    name: String,
+    description: String,
+    jurisdiction: Jurisdiction,
+    stateCode: String? = null,
+    countyName: String? = null,
+    val senders: List<Sender> = emptyList(),
+    val receivers: List<Receiver> = emptyList(),
```

Comment:
It may be good to put the `receiving-application` and `receiving-facility` in organization as well. These don't really change per schema that I've seen so far, and it could help us minimize copy-pasting schemas just to default those values.